src/applications/check-in/day-of/pages/TravelReview.jsx

```diff
@@ -28,12 +35,26 @@ const TravelQuestion = props => {
   };
   const validation = () => {
     if (agree) {
+      recordEvent({
```

Comment:
these events are already firing in the TravelPage component. Adding here adds duplicate calls.

Reply:
Thanks good catch!